# Maps configuration file
# Defines three maps with different parameters

# Load priority: true = load from files first, false = use config to create maps first
# Set to true if you want to load existing .pgm and .yaml files from the maps/ directory
# Set to false if you want to create maps from config and obstacles_config.yaml
load_from_files_priority: false

maps:
  map1:
    name: "map1"
    resolution: 0.05  # 5cm per cell
    width: 3.2       # 320 cm 
    height: 6.0      # 600 cm 
    origin: [0.0, 0.0, 0.0]  # [x, y, theta] in meters and radians
    
  map2:
    name: "map2"
    resolution: 1.2   # 120cm per cell
    width: 4.8       # 480 cm 
    height: 3.6      # 360 cm 
    origin: [3.2, 1.2, 0.0]  # Offset to the right
    
  map3:
    name: "map3"
    resolution: 0.05   # 5cm per cell
    width: 4.0       # 400 cm 
    height: 6.0      # 600 cm 
    origin: [8.0, 0.0, 0.0]  # Offset upward
